ios/RNCUIWebView.m

```diff
@@ -246,7 +246,7 @@ - (BOOL)webView:(__unused UIWebView *)webView shouldStartLoadWithRequest:(NSURLR
       @"data": data,
     }];
 
-    NSString *source = @"document.dispatchEvent(new MessageEvent('message:received'));";
+    NSString *source = [NSString stringWithFormat:@"window.@.messageReceived();", MessageHandlerName];
```

Comment:
This makes a lot more sense than the previous implementation :)

Here, within native code, window.ReactNativeWebview.messageReceived is called inside the WebView to let it know we have finished processing a message.

ps I do wonder how much time we should invest in this particular WebView; it's supposed to be replaced with WKWebView, I think.

@Titozzz?

Reply:
Yeah we will probably drop UIWebview soon. I've wanted to do it for so long anyway 😢